src-plugins/manualRegistration/manualRegistrationToolBox.cpp

```diff
@@ -331,9 +328,11 @@ void manualRegistrationToolBox::computeRegistration()
     d->process->setMovingInput(qobject_cast<medAbstractLayeredView*>(d->rightContainer->view())->layerData(0));
     d->process->update(itkProcessRegistration::FLOAT);
 
-    if(this->parentToolBox()) //if in Registration Workspace
+    medRegistrationSelectorToolBox* toolbox = dynamic_cast<medRegistrationSelectorToolBox*>(selectorToolBox());
+
+    if(toolbox) //if in Registration Workspace
```

Comment:
Perhaps we can emphasize the pipeline stuff with something like "toolbox empty in pipelines and not registration workspace", something like that :)